scripts/prepublish.js

```diff
@@ -7,9 +7,13 @@ let buildify = require('buildify');
 let rimraf = require('rimraf');
 let path = 'resources/js';
 
-if (fs.statSync(path).isDirectory()) {
+console.log('Start Preprocess for npm publishing')
+
+if (fs.existsSync(path)) {
```

Comment:
This appears to work in spite of no require('fs') but it might be worth adding while we're in here.

`let fs = require('fs');`

Reply:
@jwaspin The PR isn't showing the full file. The require is on line 5. https://github.com/adiwg/mdCodes/blob/update/npm-version-2-8-4/scripts/prepublish.js#L5

Reply:
Yeah, I guess it would help if I expanded the file.